### DISCUSSION TITLE
How can I suppress the popup wiindow when layers were added in map

### DISCUSSION THREAD

**Discussion by suredream** [opengeos/leafmap#665] [Q&A] [answered]
A bit annoying when I add more layers there
<img width="268" alt="image" src="https://github.com/opengeos/leafmap/assets/1661077/04f3f711-7e9b-40bd-9452-89c756208622">

**Comment by giswqs** [comment, marked as answer]
`info_mode=None`

![image](https://github.com/opengeos/leafmap/assets/5016453/fbbaf45d-b137-494f-bebb-74f7aa4677c4)
